src/libs/ajax/ajax-common.ts

```diff
@@ -242,4 +242,6 @@ export const fetchWDS = (wdsProxyUrlRoot) =>
   _.flow(withUrlPrefix(`${wdsProxyUrlRoot}/`), withRetryAfterReloadingExpiredAuthToken)(fetchOk);
 
 export const fetchCbas = (cbasProxyUrlRoot) =>
-  _.flow(withUrlPrefix(`${cbasProxyUrlRoot}/`), withRetryAfterReloadingExpiredAuthToken)(fetchOk);
+  _.flow(withUrlPrefix(`${cbasProxyUrlRoot}/api/batch/v1/`), withRetryAfterReloadingExpiredAuthToken)(fetchOk);
```

Comment:
Based on other services in here, it seems the pattern is to have only the root inside the `withUrlPrefix` and then add the remaining path in the caller. So which ever API is using `fetchCbas` they would append the `/api/batch/v1` as needed.